Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformModeBus.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+#include <WhiteBox/WhiteBoxToolApi.h>
+
+namespace WhiteBox
+{
+    enum class TransformType
```

Comment:
suggestion: API comment for this type would be useful

Reply:
This was pre-existing, I just moved it :) Adding a comment.

Reply:
Awesome thanks!